### PR TITLE
Fixes small typos to PT-BR homepage

### PR DESCRIPTION
**Pull request by erickpatrick**
There were a few typos that didn't break the meaning but were not nice to see

### CODE REVIEW

**Comment by developit** [comment]
Thanks @erickpatrick & @joaolucasl!

**Comment by erickpatrick** [comment]
You're welcome @developit =)

File: content/lang/pt-br/index.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ toc: false
     <h2>Produtividade Instantânea</h2>
     
     <p>
-        O "leve" fica muito mais divertido quando você não tem que sacrificar produtividade para alcancar. Preact te torna produtivo imediatamente. Tem até mesmo alguns recursos bônus:
+        O "leve" fica muito mais divertido quando você não tem que sacrificar produtividade para alcancá-lo. Preact te torna produtivo imediatamente. Tem até mesmo alguns recursos bônus:
```
Comment (guilhermedecampo) [inline content/lang/pt-br/index.md]: Should be `alcança-lo` sorry the delay in the review.

Comment (developit) [inline content/lang/pt-br/index.md]: fixed :)

**Pull request merged**